src/wp-includes/media.php

```diff
@@ -5456,6 +5460,21 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	// Conditionally skip lazy-loading on images before the loop.
+	if (
+		( 'wp_get_attachment_image' === $context || 'the_post_thumbnail' === $context )
```

Comment:
As these same values are used here, I think this should be an array - https://github.com/WordPress/wordpress-develop/pull/4300/files#diff-4d28b6652f0dab208c19aa45e1cf09f3c967060d8cc77502bfcd473a3216b612R5515

Reply:
@spacedmonkey Please see https://github.com/WordPress/wordpress-develop/pull/4300#discussion_r1196900653 for why we shouldn't do that.